cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -108,6 +108,9 @@ SECTIONS
     /* stack section */
     .stack (NOLOAD):
     {
+        KEEP (*(.puf_stack))
+        . = ALIGN(8);
+        _pstack = .;
```

Comment:
I don't see why we need to introduce this. There's no clear use case and the two consecutive aligns are redundant so this will always be equivalent to _sstack